orca-keel/src/main/kotlin/com/netflix/spinnaker/orca/keel/task/ImportDeliveryConfigTask.kt

```diff
@@ -78,8 +77,6 @@ constructor(
       keelService.publishDeliveryConfig(deliveryConfig)
 
       TaskResult.builder(ExecutionStatus.SUCCEEDED).context(emptyMap<String, Any?>()).build()
-    } catch (e: RetrofitError) {
-      handleRetryableFailures(e, context)
     } catch (e: SpinnakerServerException) {
       handleRetryableFailures(e, context)
```

Comment:
this method is deleted below. Is there a similar method available elsewhere?

Reply:
There are 2 APIs invoked inside the try block:
1. Keel Service API - keelService.publishDeliveryConfig() : With the addition of the commit - https://github.com/spinnaker/orca/commit/c41792295d4b2bba289ab131699cc7b061ec627e, the keelService uses SpinnakerRetrofitErrorHandler.
2. Igor Service API - scmService.getDeliveryConfigManifest() : With the addition of the commit - https://github.com/spinnaker/orca/commit/d430b75efeced7eb1f8d64eff6f0a160210eb2e6, the igor service uses SpinnakerRetrofitErrorHandler.

and hence with the above mentioned commits the catch block with RetrofitError and the handler logics associated with it becomes irrelevant.